static/etc/system-image/writable-paths

```diff
@@ -65,3 +65,5 @@
 /etc/machine-id                         auto                    persistent  transition  none
 # make update-motd writable for branding purposes
 /etc/update-motd.d                      auto                    persistent  transition  none
+# for systemd-tmpfiles configurations
+/etc/tmpfiles.d                         auto                    persistent  transition  none
```

Comment:
Nitpick: this is complaining about missing newline at EOF

Reply:
thanks - I've just fixed this up and force-pushed.